pkg/chains/formats/slsa/attest/attest.go

```diff
@@ -108,15 +113,76 @@ func Invocation(obj objects.TektonObject, params []v1beta1.Param, paramSpecs []v
 	return i
 }
 
-func convertConfigSource(source *v1beta1.RefSource) slsa.ConfigSource {
-	if source == nil {
-		return slsa.ConfigSource{}
+func StepV1Beta1(step *v1beta1.Step, stepState *v1beta1.StepState) StepAttestation {
```

Comment:
Instead of having separate v1 and v1beta1 extraction functions, can we convert as needed to feed into the same func?

Reply:
The only place this may get a little tricky is for pipeline resource support, but for general provenance extraction this will likely be easier to maintain and make sure there's no skew.

Reply:
With splitting this PR into migrate internal logic to use v1 + convert to v1beta1 for backwards compat and a future PR to add a Tekton v1 API format (vs v1beta1) v2alpha3 this generic function is not needed in this PR (only v1beta1 will suffice as that is the only objects passed to format/ currently given the backwards compat).  In the PR adding v2alpha3 (https://github.com/tektoncd/chains/issues/1026) I will adhere to the advice here.  Marking resolved